applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/boards/nrf54l15pdk_nrf54l15_cpuapp_0_2_0.overlay

```diff
@@ -10,54 +10,98 @@
 };
 
 / {
-	/* Disable leds and redefine them to align configuration with CAF LEDs requirements.
-	 * The configuration needs to match the used board revision (v0.2.0).
-	 */
-	leds {
-		status = "disabled";
+	pwmleds0 {
```

Comment:
these changes seem to overlap with https://github.com/nrfconnect/sdk-zephyr/pull/1661/files . Maybe they should be a part of common board definition? cc: @kl-cruz

Reply:
I have mixed feelings about this (e.g. we use another GPIO pin assignments in nRF Desktop). If this would be added to board as "default" we would need to redefine it in application's overlay anyway I think.